.github/workflows/validation.yml

```diff
@@ -21,23 +21,6 @@ jobs:
       - name: Download packages
         run: yarn
 
-  audit:
```

Comment:
I still don't like I had to remove this, but hope storybook updates their dependencies so I can add it back.